src/Utilities/NewTimer.cpp

```diff
@@ -318,9 +320,14 @@ TimerManagerClass::print_stack(Communicate* comm)
       max_name_len = std::max(name_len, max_name_len);
     }
 
+    char tmpout[256];
     std::string timer_name;
     pad_string("Timer", timer_name, max_name_len);
-    printf("%s  %-9s  %-9s  %-10s  %-13s\n",timer_name.c_str(),"Inclusive_time","Exclusive_time","Calls","Time_per_call");
+
+    snprintf(tmpout, 255, "%s  %-9s  %-9s  %-10s  %-13s\n",timer_name.c_str(),"Inclusive_time","Exclusive_time","Calls","Time_per_call");
+    tmpout[255] = '\0';
```

Comment:
A terminating null character is automatically appended after the content written.
Please change all the 255 to 256 because snprintf only writes n-1+"\0" characters.
Found in http://www.cplusplus.com/reference/cstdio/snprintf/